opentelekomcloud/import_opentelekomcloud_sfs_file_system_v2_test.go

```diff
@@ -0,0 +1,29 @@
+package opentelekomcloud
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+// PASS
```

Comment:
no need to have such test result "//PASS"

Reply:
Done